### PR TITLE
[ci skip] Readd "Use" Javadoc tab thingy

### PR DESCRIPTION
**Pull request by emilyy-dev**
See `-use` in https://docs.oracle.com/en/java/javase/18/docs/specs/man/javadoc.html#standard-options-for-the-standard-doclet

### CODE REVIEW

**Comment by lynxplay** [comment]
Might want to be a `[ci skip]` commit ? 
You'd have to wait for the next PR to be merged for it to go into effect, but I unsure if this warrants a new build to be published to the downloads API. Tho I am sure one of the core team/dev peeps have a stronger opinion for one of the two options :+1: 

Beyond that, lgtm.

**Review by NoahvdAa — APPROVED** [review]

**Comment by JRoy** [comment]
ci-skip please

**Comment by emilyy-dev** [comment]
Donio 👍

**Review by kennytv — APPROVED** [review]

**Review by JRoy — APPROVED** [review]

**Pull request merged**
